Source/Urho3D/Engine/Engine.cpp

```diff
@@ -260,13 +286,13 @@ bool Engine::Initialize(const StringVariantMap& parameters)
     GetSubsystem<Time>()->SetTimerPeriod(1);
 
     // Configure max FPS
-    if (GetParameter(EP_FRAME_LIMITER, true) == false)
+    if (GetParameter(EP_FRAME_LIMITER) == false)
```

Comment:
Why change defaults here and in other places? We still want sane defaults if setting is not specified and not set in config or config does not even exist.